docs/QuickStart.md

```diff
@@ -0,0 +1,159 @@
+# QuickStart
+
+## Installation
+
+Two ways for installation: you can get NNI from pypi or clone the repo and install it from there.
+
+**Install through pip**
+
+* We support Linux and MacOS in current stage, Ubuntu 16.04 or higher, along with MacOS 10.14.1 are tested and supported. Simply run the following `pip install` in an environment that has `python >= 3.5`.
+
+```bash
+    python3 -m pip install --user --upgrade nni
+```
+
+* Note:
+  * If you are in docker container (as root), please remove `--user` from the installation command.
+  * If there is any error like `Segmentation fault`, please refer to [FAQ][1]
+
+**Install through source code**
+
+* We support Linux (Ubuntu 16.04 or higher), MacOS (10.14.1) in our current stage. 
+* Run the following commands in an environment that has `python >= 3.5`, `git` and `wget`.
+
+```bash
+    git clone -b v0.4.1 https://github.com/Microsoft/nni.git
+    cd nni
+    source install.sh
+```
+
+For the system requirements of NNI, please refer to [Install NNI][2]
+
+## First "Hello World" experiment: MNIST
+
+Now let's start to run our first MNIST experiment using NNI.
+
+Here is a configuration file in NNI experiment:
+
+```yaml
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+trainingServicePlatform: local
+searchSpacePath: search_space.json
+useAnnotation: false
+tuner:
+  builtinTunerName: TPE
+  classArgs:
+    optimize_mode: maximize
+trial:
+  command: python3 mnist.py
+  codeDir: .
+  gpuNum: 0
+```
+
+**We did two things here:**
+
+* specify the `range of hyper-paramaters` to be searched for this experiment in `search_space.json`.
```

Comment:
I think we should tell what's the target here and how to do next.

Reply:
I don't understand your current problem clearly, but I made a change to make it more understandable. What do you think now?